src/main/resources/resources/logisim/strings/analyze/analyze_pt.properties

```diff
@@ -120,6 +120,7 @@ openErrorTitle = Arquivo de leitura de erros
 #
 karnaughNoInputsError = Não há entradas disponíveis.
 karnaughNoOutputError = Nenhuma saída selecionada.
+# ==> karnaughTooFewInputsError =
```

Comment:
[DeepL](https://www.deepl.com/translator) suggests: "Entradas demasiado poucas para a tabela."

Reply:
Added in commit fa06451